app/views/hyrax/file_sets/media_display/_image.html.erb

```diff
@@ -0,0 +1,44 @@
+<%
```

Comment:
Do we need to do all this here, or is the [representative media partial](https://github.com/samvera/hyrax/blob/master/app/views/hyrax/base/_representative_media.html.erb) sufficient?

Reply:
It does seem like we need to override the default behavior of mage display for IIIF. I think we can move the codes for the iiif_url to a helper class instead, otherwise the override of the Hyrax::FileSetsControl for a custom FileSetPresenter with the function to generate the iiif_url for the image. What do you think?

Reply:
But isn't that partial for the display of an image within the context of the fileset section of the view, not the primary representative media for the object show page? I would think the latter is presumably the context in which we'd want to present a IIIF viewer?

Or maybe put another way, if we don't change the representative media partial, are we going to be left with a static image display, but have the IIIF viewer elsewhere, displayed in the FileSet context? That seems like it might be odd.

Reply:
Right, it's the FileSet section of the viewer that could serve the purpose of testing the functionality of the IIIF and Cantaloupe. We can added it to the object viewer, but it looks like we need some UI works for the object viewer ... Should we make the changes to the object viewer now or later when we get some inputs?

Reply:
@mcritchlow How about making a similar change as for the fileset now, and we adjust the object viewer later once we get some Inputs?

Reply:
Do you mean make the change in both places? If so, I'm not sure we need that for this R&D. I guess for now it's more important to try things out than have it done perfectly the first time, as long as we can revert/replace as needed. Any thoughts on your end @VivianChu ?

Also, @lsitu , what are your thoughts about using [the gem I referenced](https://github.com/ucsdlib/horton/pull/137#issuecomment-327883273)? Might make it easier to use and potentially remove if we decide to go with, say, universal viewer instead in the future?

Reply:
@mcritchlow I think it's done. See my comment earlier, https://github.com/ucsdlib/horton/pull/137#issuecomment-328136108.